docs/REFERENCE.md

```diff
@@ -88,20 +88,20 @@ The `files` block configures the file serving.
 
 ### SPA Block
 
-The `spa` block configures the Web serving for SPA assets.
+The `spa` blocks configures the Web serving for SPA assets. Can be defined multiple times as long as the `base_path` is unique.
```

Comment:
https://github.com/avenga/couper/blob/master/docs/REFERENCE.md#server-block:
In `server`'s "Nested Block(s)": add "(s)" to "SPA Block".